apps/health_alerts/templates/health_alerts/health_alert_index_page.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block title_suffix %}
+  {{ page.title }}
+{% endblock %}
```

Comment:
Naming the title of the health alerts index page `Health Alerts` seems to make the title of the page `Health Alerts Health Alerts`